src/open_inwoner/openzaak/api_models.py

```diff
@@ -79,8 +80,11 @@ def process_data(self) -> dict:
             "start_date": self.startdatum,
             "end_date": getattr(self, "einddatum", None),
             "description": self.zaaktype.omschrijving,
-            "current_status": status_translate.from_glom(
-                self, "status.statustype.omschrijving", default=""
+            "current_status": (
+                glom(self, "status.statustype.statustekst")
+                or status_translate.from_glom(
+                    self, "status.statustype.omschrijving", default=""
+                )
```

Comment:
I forgot to add a note that that this relates to earlier about modifying/expanding the `from_glom()` method